frontend/src/scenes/instance/Licenses/index.tsx

```diff
@@ -8,66 +9,175 @@ import { PageHeader } from 'lib/components/PageHeader'
-    },
-    {
-        title: 'Plan',
-        dataIndex: 'plan',
-    },
-    {
-        title: function Render() {
-            return (
-                <Tooltip
-                    placement="right"
-                    title="Maximum number of team members that you can have across all organizations with your current license."
-                >
-                    Max # of team members
-                    <InfoCircleOutlined className="info-indicator" />
-                </Tooltip>
-            )
+        {
+            title: 'Plan',
+            dataIndex: 'plan',
         },
-        render: function renderMaxUsers(license: any) {
-            return license.max_users === null ? 'Unlimited' : license.max_users
+        {
+            title: function Render() {
+                return (
+                    <Tooltip
+                        placement="right"
+                        title="Maximum number of team members that you can have across all organizations with your current license."
+                    >
+                        Max # of team members
+                        <InfoCircleOutlined className="info-indicator" />
+                    </Tooltip>
+                )
+            },
+            render: function renderMaxUsers(_, license: LicenseType) {
+                return license.max_users === null ? 'Unlimited' : license.max_users
+            },
         },
-    },
-    {
-        title: 'Key',
-        render: function renderActive(license: any) {
-            return <CodeSnippet>{license.key}</CodeSnippet>
+        {
+            title: 'Key',
+            render: function renderActive(_, license: LicenseType) {
+                return <CodeSnippet>{license.key}</CodeSnippet>
+            },
         },
-    },
-    {
-        title: 'License added on',
-        render: function renderActive(license: any) {
-            return humanFriendlyDetailedTime(license.created_at)
+        {
+            title: 'License added on',
+            render: function renderActive(_, license: LicenseType) {
+                return humanFriendlyDetailedTime(license.created_at)
+            },
         },
-    },
-]
-
-export function Licenses(): JSX.Element {
-    const [form] = Form.useForm()
-    const { licenses, licensesLoading, error } = useValues(licenseLogic)
-    const { createLicense } = useActions(licenseLogic)
+        {
+            width: 0,
+            render: function renderActive(_, license: LicenseType) {
+                return (
+                    <More
+                        overlay={
+                            <>
+                                <LemonButton
+                                    type="stealth"
+                                    style={{ color: 'var(--danger)' }}
+                                    onClick={() => setShowConfirmCancel(license)}
+                                    fullWidth
+                                >
+                                    Deactivate license
```

Comment:
If someone just wanted to delete a license that expired, they'd get the scary "Are you sure" prompt, when in that case there's zero impact. Could we instead just say "Delete license" and not ask for confirmation then? This may be someone who churned, but also someone who upgraded from Scale to Enterprise a while ago, so it might be better to minimize fear in such case.

Reply:
I'd rather not write extra logic to delete expired licenses so I've just disabled the option to remove.